src/cli/cmds/pod.toit

```diff
@@ -152,7 +152,7 @@ create-pod-commands -> List:
       --run=:: list it
   cmd.add list-cmd
 
-  print-cmd := Command "print"
+  print-spec-cmd := Command "print-spec"
```

Comment:
Not sure I'm crazy about the spec abbreviation here. We typically call these specifications, so maybe having 'print-spec' as an alias and 'print-specification' as the main thing would be nicer.